src/column.rs

```diff
@@ -395,10 +423,18 @@ impl Column {
 					tables.index.validate_plan(record.index, log)?;
 				} else if let Some(table) = reindex.queue.iter().find(|r|r.id == record.table) {
 					table.validate_plan(record.index, log)?;
-				}
-				else {
+				} else {
+					// [Review] could this case be handle by triggering reindex in 'replay_all_logs',
+					// or is it specific to validate?
+
 					// Re-launch previously started reindex
 					// TODO: add explicit log records for reindexing events.
+
+					// [Review] I did think of writing it in log, but then conclude it would be wrong
+					// because reindex is triggered after enact_log. (so would need specific log).
+					// Would maintaining a persistence of 'index_bits' be enough (so we restart reindex at start)
+					// when opening index tables.
```

Comment:
Maintaining it where? We'd need to provide the same consistency guarantees as we do for log records.